src/logging/zlog/inc/zlog-config.h

```diff
@@ -15,8 +15,8 @@
 // enabling this will slow down the log
 // #define ZLOG_FORCE_FLUSH_BUFFER
 
-#define ZLOG_BUFFER_LINE_MAXCHARS 512
```

Comment:
add comment as to how these values were derived so that someone doesnt come alone later and say "these are too small! I'm going to change MAXLINES to 1024"